py/picca/tests/delta_extraction/mask_tests.py

```diff
@@ -114,24 +117,369 @@ def test_absorber_mask(self):
         reset_logger()
         self.compare_ascii(test_file, out_file)
 
-    def test_dla_mask(self):
+    def test_bal_mask_ai_remove(self):
```

Comment:
the added tests should only add to existing tests, so better to have them added. Did not check all the internals...

Reply:
I added them, I just renamed the `test_dla_mask` to `test_dla_remove` that's why this line is flagged as removed